README.md

```diff
@@ -60,7 +60,19 @@ listening for API requests and provisioning volumes accordingly.
 
 ### Architecture and Operation
 
-The PMEM-CSI driver can operate in two different DeviceModes: LVM and Direct
+The PMEM-CSI driver can operate in two different DeviceModes: LVM and Direct. This tables describes shows overview and comparison of those modes. There is more detailed modes explanation down in following paragraphs.
```

Comment:
This is the first time that we use the special "DeviceMode" spelling. We either need to explain why it is spelled this way or just spell it as "device modes".

As it stands, it looks like a special term that readers are supposed to be familiar with.

Reply:
"this tables describes shows overview and comparison" -> "this table contains an overview and comparison"